test/Dialect/Arc/basic-errors.mlir

```diff
@@ -548,3 +548,12 @@ hw.module @vectorize(in %in0: i4, in %in1: i4, out out0: i4) {
 
 // expected-error @below {{state type must have a known bit width}}
 func.func @InvalidStateType(%arg0: !arc.state<index>)
+
+// -----
+
+// expected-note @below {{prior use here}}
+hw.module @assert(in %in : i4) {
+  // expected-error @below {{use of value '%in' expects different type than prior uses: 'i1' vs 'i4'}}
+  arc.assert %in, hw.constant true
```

Comment:
What does the `hw.constant true` do here? Can we remove it?

Reply:
Yes. We can. I just provided a constant `i1` parameter.